.github/workflows/auto-merge.yml

```
@@ -0,0 +1,35 @@
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)